include/ceed/backend.h

```diff
@@ -404,6 +406,7 @@ CEED_EXTERN int CeedQFunctionAssemblyDataSetObjects(CeedQFunctionAssemblyData da
 CEED_EXTERN int CeedQFunctionAssemblyDataGetObjects(CeedQFunctionAssemblyData data, CeedVector *vec, CeedElemRestriction *rstr);
 CEED_EXTERN int CeedQFunctionAssemblyDataDestroy(CeedQFunctionAssemblyData *data);
 
+CEED_EXTERN int CeedOperatorGetOperatorAssemblyData(CeedOperator op, CeedOperatorAssemblyData *data);
```

Comment:
I meant this one, sorry. You moved the function definition

Reply:
Yeah this is now consistent with `ceed-preconditioning.c` where we define `GetOperatorAssemblyData` right before `OperatorAssemblyDataCreate`.